src/blant.c

```diff
@@ -707,20 +729,76 @@ int RunBlantFromEdgeList(int k, int numSamples, int numNodes, int numEdges, int
     return RunBlantInThreads(k, numSamples, G);
 }
 
+void buildGraphlets(GRAPH* G, SET* prev_nodes, int numSamplesPerNode, int *tempCountPtr) {
```

Comment:
This function needs a more descriptive name, and some comments above it describing under what circumstances it's called, what the return values are, etc. I think it's only called when the sampling method is SEEDS, right? (the old -w1 method). So please describe that, and also what that last tempCountPtr parameter is for. Finally, the standard in BLANT is that function names begin with a capital letter. Is there a good reason this function diverts form that standard?

Reply:
I just changed the function name to BuildGraphletsAsSeeds. I think it might be a more appropriate name than the previous name buildTGraphlets because the function is building graphlets and the graphlets built are sampled as seeds.